api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -52,6 +53,10 @@ message IPTagging {
     repeated config.core.v3.CidrRange ip_list = 2;
   }
 
+  // optional header to use for ip-tagging instead of default
```

Comment:
style-nit: sentence starts with capital letter and ends with '.'.

I suggest explicitly writing the name of the default header name.

Reply:
How silly of me, apologies!